infra/compile.py

```diff
@@ -157,9 +161,20 @@ def check_args(args, ctx):
     # skia.Font
     "getMetrics",
 
-    # skia.Rect
+    # skia.*Rect
     "MakeEmpty",
     "MakeLTRB",
+    "MakeXYWH",
+    "left",
+    "top",
+    "bottom",
+    "width",
+    "height",
+    "isEmpty",
+    "roundOut",
+    "intersects",
+    "Intersects",
```

Comment:
Is there really an `Intersects` method?

Reply:
Yes. Skia.Rect.Intersects (static function)

There is no "intersects" though, removed that.